src/components/Form/components/OrInput.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import FileInput from './FileInput';
+import './OrInputStyle.css';
+
+export const OrInput = ({
+  uploadedDocuments,
+  documentsToUpload,
+  documentType,
+  disabled,
+  onChange,
+  children,
+  label,
+}) => {
+  return (
+    <div id="or-input-form" className="form__group">
+      <div>{children}</div>
+      <div className="separator">
+        <span>ou</span>
+      </div>
+      <div>
+        <FileInput
+          label={label}
```

Comment:
for simplicity, maintainability and readability I'd rather use props splating here

```
export const OrInput = ({
  children,
  ...props,
}) => {
```

then 

```suggestion
        <FileInput
          ...props
```

Reply:
or maybe there is a way to use 2 childrens and inserting the or between them ?